src/inspector/inspectorhandler.cpp

```diff
@@ -57,6 +60,35 @@ InspectorItemPicker* s_itemPicker = nullptr;
 
 std::function<void(InspectorHandler* inspectorHandler)> s_constructorCallback;
 InspectorHotreloader* s_hotreloader = nullptr;
+
+QJsonValue qjsValueToQJsonValue(const QJSValue& qjsValue) {
```

Comment:
This function will only ever be used in the tests themselves, so I don't think it's that important to add tests for it. The result of this failing is whatever value we are expecting is a test does not come in as expected. It's extremely unlikely a developer does not notice this at the time of the implementation of the test... Let me know what you think.

Reply:
I don't understand why this will only be used by the tests themselves, and how it might not be used at some point in the future in non-test code. (Though, I don't have the best understanding of the inspector code.) Is that what you're saying? Or just that it's only used by tests right now? I think if it's possible some future engineer is going to use this code for something other than tests, we should add a unit test here.

Reply:
The inspector is only used by our debugging tools and tests. This is inside an anonymous namespace inside the inspector module, therefore it cannot be imported by other modules.